dbschema/seeds/011.product.edgeql

```diff
@@ -0,0 +1,1080 @@
+              }
+            }
+          ]
+        },
+      "steps": [
+          "CommunityTesting",
+          "BackTranslation",
+          "ConsultantCheck",
+          "Completed"
+        ],
+      "methodology": "Paratext",
+      "describeCompletion": "Translate and print trial copies of Jeremiah",
+      "pnpIndex": 24,
+      "progressTarget": 100,
+      "progressStepMeasurement":"Percent",
+      "totalVerses": 1364,
+      "totalVerseEquivalents": 2131
+     },
+     {
+      "project": "Misty Mountains",
+      "scripture":
+        {
+          "label": "Zechariah",
+          "verses": [
+            {
+              "label": "Zechariah",
+              "end": {
+                "book": "Zechariah",
+                "chapter": 14,
+                "verse": 21,
+                "verseId": 37
+              },
+              "start": {
+                "book": "Zechariah",
+                "chapter": 1,
+                "verse": 1,
+                "verseId": 38
+              }
+            }
+          ]
+        },
+      "steps": [
+          "ExegesisAndFirstDraft",
+          "TeamCheck",
+          "BackTranslation",
+          "ConsultantCheck"
+        ],
+      "methodology": "Paratext",
+      "pnpIndex": 38,
+      "progressTarget": 100,
+      "progressStepMeasurement":"Percent",
+      "totalVerses": 211,
+      "totalVerseEquivalents": 330
+     },
+     {
+      "project": "Misty Mountains",
+      "scripture":
+        {
+          "label": "Exodus",
+          "verses": [
+            {
+              "label": "Exodus",
+              "end": {
+                "book": "Exodus",
+                "chapter": 40,
+                "verse": 38,
+                "verseId": 39
+              },
+              "start": {
+                "book": "Exodus",
+                "chapter": 1,
+                "verse": 1,
+                "verseId": 40
+              }
+            }
+          ]
+        },
+      "steps": [
+          "ExegesisAndFirstDraft",
+          "TeamCheck",
+          "CommunityTesting",
+          "BackTranslation",
+          "ConsultantCheck",
+          "Completed"
+        ],
+      "methodology": "Paratext",
+      "describeCompletion": "Printed trial editions and audio recordings of Exodus.",
+      "pnpIndex": 4,
+      "progressTarget": 100,
+      "progressStepMeasurement":"Percent",
+      "totalVerses": 1213,
+      "totalVerseEquivalents": 1213
+     },
+     {
+      "project": "Lothlorien",
+      "scripture":
+        {
+          "label": "Song of Solomon",
+          "verses": [
+            {
+              "label": "Song of Solomon",
+              "end": {
+                "book": "Song of Solomon",
+                "chapter": 8,
+                "verse": 14,
+                "verseId": 41
+              },
+              "start": {
+                "book": "Song of Solomon",
+                "chapter": 1,
+                "verse": 1,
+                "verseId": 42
```

Comment:
This start verse ID needs be less than the end ID. Would be worth double checking other places.
You check by running
```edgeql
select Scripture::Collection {*}
```
and seeing if it gives you an error or not.

You can also
```edgeql
delete Product;
```
to be able to re-run the seed script for changes you make.

Reply:
@CarsonF , good catch here.  I was getting an error and couldn't figure out why but this makes total sense now.

Reply:
Ironically Bryan suggested that I write a constraint for this. But I declined because it would be a non trivial trigger assertion and I said that these inserts would be constructed in an app code abstraction. But here we are. So Bryan gets a point 😁